packages/klevu-core/src/store/lastClickedProducts.ts

```diff
@@ -147,7 +147,8 @@ class LastClickedProducts {
       }
     }
 
-    const itemsToTake = Math.floor(currentClicks.length / 3) * 3
+    let itemsToTake = Math.floor(currentClicks.length / 3) * 3;
+    if(itemsToTake>18) itemsToTake = 18;
```

Comment:
pls move 18 to a constant in this file and use constant here, so the name of constant will be helpful to understand what this number is
suggestion: `MAX_PERSONALISED_RESULTS_FOR_SEARCH`